.buildkite/pipeline.yml

```diff
@@ -130,6 +130,15 @@
       --test_tag_filters=-flaky
       -- //:all -//:core_worker_test
 
+- label: ":cpp: Tests (UBSAN)"
```

Comment:
If running ubsan separately has too much (testing / maintenance) overhead, we can enable ubsan together with asan too.